cpp/src/io/orc/stripe_data.cu

```diff
@@ -1602,7 +1756,8 @@ CUDF_KERNEL void __launch_bounds__(block_size)
         if (is_rlev1(s->chunk.encoding_kind)) {
           numvals = Integer_RLEv1<int64_t>(bs, &s->u.rlev1, s->vals.i64, numvals, t);
```

Comment:
Update: Figured out how to generate RLEv1 data. Working on creating the bite-size sample and applying the fix... :car:

Reply:
Update: Generating bite-size RLEv1 data with user-defined run lengths turns out to be a non-trivial task for my custom decoder. But if my understanding of the spec is not incorrect, RLEv1 TIMESTAMP  is actually not subject to the same bug as RLEv2 TIMESTAMP  does. This is because in RLEv1 the run length is represented by 7 bits for both the "runs" and the "literals", which is 127 instead of 512 as in RLEv2. The decoder can then always consume enough runs from the SECONDARY stream such that its progress is always ahead of the DATA stream's, not the opposite.